mcs/class/System.Messaging/System.Messaging/MessageQueueInstaller.cs

```diff
@@ -27,11 +27,14 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
+#if !XAMMAC_4_5
 using System;
```

Comment:
Why do you need the additional `#if !NO_SYSTEM_DRAWING_DEPENDENCY` further down in this file if you're already excluding the entire file from the XM 4.5 build here?